WorkflowExample/WorkflowUIKitTests/UIKitConsumerTests.swift

```diff
@@ -1,5 +1,5 @@
 //
-//  UIKitPresenterTests.swift
+//  UIKitConsumerTests.swift
```

Comment:
This file is HUGE, it'd be a good idea to break the tests into different logical file groups